bookdown/06-extending-tuner.Rmd

```diff
@@ -0,0 +1,65 @@
+## Adding new Tuners {#extending-tuners}
+
+In this section, we show how to implement a custom tuner for `r mlr_pkg("mlr3tuning")`.
+
+### Adding a new Tuner via bbotk
+
+The `r mlr_pkg("bbotk")` package provides black box optimization algorithms.
+If the new custom tuner is not restricted to hyperparameter tuning, it should be implemented in `r mlr_pkg("bbotk")`.
+The `bbotk::Optimizer` can be easily transformed to a `mlr3tuning::Tuner` later.
+
+1. Check the tuner does not already exist as a `bbotk::Optimizer` or `mlr3tuning::Tuner`.
```

Comment:
Maybe a small point but is it obvious to a user how they can check this? Like is there specific listing functionality or optimizers or should they check GH?

Reply:
Added links to gh repositories.